### PR TITLE
Add-description-field-to-Place-category

### PR DESCRIPTION
**Pull request by javasia**

### CODE REVIEW

**Comment by javasia** [comment]
[1SP Add "description" field to Place category (front + back)](https://trello.com/c/0gcPnmzV/144-1sp-add-description-field-to-place-category-front-back)

File: frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Name.js [hunk 1]
```diff
@@ -22,50 +22,53 @@ const styles = theme => ({
...
+    const {classes, name} = this.props
 
     return (
       <form className={classes.container} noValidate autoComplete="off">
         <TextField
           disabled = {this.state.isDisabled}
           id="outlined-bare"
           className={classes.textField}
-          defaultValue={name}
+          defaultValue={name || "Enter name"}
```
Comment (zozich) [inline frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Name.js]: I suppose `Enter name` should go to `defaultProps`

File: frontend/admin_panel/public/index.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" /> -->
```
Comment (zozich) [inline frontend/admin_panel/public/index.html]: Why?

Comment (javasia) [reply frontend/admin_panel/public/index.html]: To aviod errors thrown to console. Just to keep it simple.

File: frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Name.js (hunk 1 above)
Comment (javasia) [reply frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Name.js]: Why not to move default values to a view?

**Pull request merged**